test/e2e-common.sh

```diff
@@ -367,10 +361,6 @@ function ingress_class() {
 
 # Uninstalls Knative Serving from the current cluster.
 function knative_teardown() {
-  if [[ -z "${INSTALL_CUSTOM_YAMLS}" && -z "${INSTALL_RELEASE_YAML}" ]]; then
```

Comment:
I would prefer keeping these checks here. When the framework (or another script) calls knative_teardown and there's nothing to uninstall, you'll get a single ">> Uninstalling Knative serving" message, which I think will be confusing.

Reply:
Overall LGTM, but looks like you missed this comment?

Reply:
Ah, good call.